src/test/kotlin/step3/fixture/CarCreateDtoFixture.kt

```diff
@@ -0,0 +1,9 @@
+package step3.fixture
+
+import step3.dto.CarCreateDto
+
+object CarCreateDtoFixture {
+    fun getMany(amount: Int): List<CarCreateDto> {
+        return (1..amount).map { CarCreateDto(name = "relkimm") }
```

Comment:
List 생성 함수로 간결하게 표현 가능하겠네요~ 참고 부탁드려요!
 
```suggestion
        return List(amount) { CarCreateDto(name = "relkimm") }
```

Reply:
요고 생각보다 유용하게 쓸 것 같더라구요😃 변경해 보았습니다~! https://github.com/next-step/kotlin-racingcar/pull/883/commits/1f0477a2c0944c50627e1428cd6c2ece4f43b4a9